### DISCUSSION TITLE
TUI InfoTec GmbH is a new company using resilience4j

### DISCUSSION THREAD

**Discussion by MarkusSchmalzried** [resilience4j/resilience4j#1832] [General]
Hi guys,

I'd like to inform you that TUI now also uses resilience4j very successfully for their primary booking backend system. I'm one of the senior developers at TUI InfoTec GmbH who introduced resilience4j in our micro service architecture.

Perhaps this is interesting for you and you wanna mention TUI in your company list:

TUI InfoTec GmbH (Backend applications inside of reservation booking workflow streams for accommodations)

**Comment by RobWin** [comment]
Thank you very much for the Information.
